databases_shiny_apps/README.md

```diff
@@ -0,0 +1,96 @@
+# Connecting R to a MS Access Database
+
+## References
+- R Studio- [Connect to a Database](https://db.rstudio.com/getting-started/connect-to-database)
+- MS Office Drivers- [Adding an ODBC data source](https://support.office.com/en-us/article/administer-odbc-data-sources-b19f856b-5b9b-48c9-8b93-07484bfab5a7)
+- RDocumentation- [DBI Package](https://www.rdocumentation.org/packages/DBI/versions/0.5-1)
+- RDocumentation- [odbc Package](https://www.rdocumentation.org/packages/odbc/versions/1.0.1)
+
+## Definitions
+**ODBC** : "Open Database Connectivity" this is a standard API (application programming interface) for accessing database management systems
+
+**DSN** : "Data Storage Name" this stores the connection information for your files in a text file not in the windows registry so that
```

Comment:
```suggestion
**ODBC**: "Open Database Connectivity" is a standard API (application programming interface) for accessing database management systems

**DSN**: "Data Storage Name" stores the connection information for your files in a text file not in the windows registry so that
```

Remove the "this" and also remove the space between the acronym and the colon. Also the DSN trails off "in a text file not in the windows registry so that", so I think this needs to get finished.